device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-D48C8/td3-a7050cx3-32s-48x50G+8x100G.config.bcm

```diff
@@ -1,3 +1,13 @@
+host_as_route_disable=1
+sai_trap_group_priority=1000
```

Comment:
looks like based on latest conversation with BCM, this attribute is not required to be set. @gechiang ?

Reply:
next release, i think.

Reply:
Yes. that is also my understanding based on yesterday's meeting with Jingping Xie of BRCM.
He mentioned that on their next 4.3 deliverables is where we need NOT set this any more and the way we configure the ACL using DROP action will achieve the outcome we need (Drop FWD path packets but allow Control packets still trapped to CPU).  There is a way to drop both control plane and forwarding place pacekts if we choose to use DENY action to drop the packet when programming the ACL.